Environment_Engine/Compute/OpeningsFromGlazingRatio.cs

```diff
@@ -0,0 +1,98 @@
+﻿using BH.Engine.Geometry;
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using BH.oM.Environment.Configuration;
+using BH.oM.Environment.Elements;
+using BH.oM.Geometry;
+using ICSharpCode.Decompiler.IL.Patterns;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+
+namespace BH.Engine.Environment
+{
+    public static partial class Compute
+    {
+        [Description("Creates openings for input Panels at the locations provided.\n The total area of the openings is equal to the total area of the external panels, multiplied by the glazing ratio.")]
+        [Input("panelsAsSpaces", "Panels as spaces - A collection of Environment Panels which will be used to identify the host panel for the opening from the provided location point.")]
+        [Input("glazingLocations", "The point in 3D space corresponding to the desired locations of the openings.")]
+        [Input("glazingRatio", "Ratio of total external panel surface area and glazed area. Ratio as decimal {0-1}.")]
+        [Input("panelsToIgnore", "Optional input for selecting a collection of Environment Panels to be ignored in the calculation")]
+        [Input("sillHeight", "Optional input for defining the distance between the base of the panel and the bottom of the opening - default: 0.5m.")]
```

Comment:
```suggestion
        [Input("sillHeight", "Optional input for defining the distance between the base of the panel and the bottom of the opening - default: 0.5.")]
```

I would recommend using an appropriate quantity attribute instead of putting `m` in the description.